Src/AutoFixture.NUnit3.UnitTest/PilotTest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class PilotTest
+    {
+        [Test]
+        public void AssertTrue()
+        {
+            Assert.IsTrue(true);
```

Comment:
This is a dummy test to make sure that NUnit3 is referenced and works correctly. However in the build process I could not find a way to create a NUnit task that supports NUnit 3. So this unit test assembly now serves only for local development, not for continuous integration (as it's excluded from xUnit assemblies - see below - but not run with a NUnit build task, as is assembly AutoFixture.NUnit2.UnitTest)